src/core/memory.cc

```diff
@@ -1554,6 +1554,10 @@ statistics stats() {
         alloc_stats::get(alloc_stats::types::foreign_mallocs), alloc_stats::get(alloc_stats::types::foreign_frees), alloc_stats::get(alloc_stats::types::foreign_cross_frees)};
 }
 
+size_t free_memory() {
```

Comment:
No declaration? How can it be used?

Reply:
Oops. Give me a second.

Reply:
Done.

Reply:
(I mean, even without the header decl you could still use it by declaring it yourself at the call site. That's what I was doing, to avoid a recompilation of Scylla.)